src/backend/utils/misc/queryjumble.c

```diff
@@ -597,6 +597,24 @@ JumbleExpr(JumbleState *jstate, Node *node)
 				JumbleExpr(jstate, rtfunc->funcexpr);
 			}
 			break;
+		case T_Integer:
+			{
+				APP_JUMB(intVal(node));
+			}
+			break;
```

Comment:
Add to the description why we need to handle this type of node and why it's implemented that way.

Reply:
Updated description with the requested information